jquery-mockjax/jquery-mockjax.d.ts

```diff
@@ -0,0 +1,43 @@
+// Type definitions for jQuery Mockjax 2.0.1
+// Project: https://github.com/jakerella/jquery-mockjax
+// Definitions by: Laszlo Jakab <https://github.com/laszlojakab/>, Vladimir Đokić <https://github.com/vladeck/>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+/// <reference path="../jquery/jquery.d.ts" />
+
+interface MockJaxSettings {
+    url?: string | RegExp;
+    data?: any;
+    type?: string;
+    headers?: any;
```

Comment:
If we are talking about optional headers, we can go even further: define it as an interface (IMockJaxSettingsHeaders for example) - so that the key/value is enforced on the user and easy to follow. But, if possible, I would leave this for further iterations.